src/nvim/option.c

```diff
@@ -1596,6 +1617,8 @@ int do_set(char_u *arg, int opt_flags)
             // origval may be freed by
             // did_set_string_option(), make a copy.
             saved_origval = (origval != NULL) ? xstrdup((char *)origval) : 0;
+            saved_origval_l = (origval_l != NULL) ? xstrdup((char *)origval_l) : 0;
+            saved_origval_g = (origval_g != NULL) ? xstrdup((char *)origval_g) : 0;
```

Comment:
Why 0 instead of NULL? Compiler can replace `0`, `NULL` with the correct value but this variable is a pointer.

Reply:
Nvrmnd. didn't notice the lines above. Still, that's confusing to me.